pkg/storkctl/migrationschedule.go

```diff
@@ -162,17 +233,28 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 		},
 	}
 	createMigrationScheduleCommand.Flags().StringSliceVarP(&namespaceList, "namespaces", "", nil, "Specify the comma-separated list of namespaces to be included in the migration")
+	createMigrationScheduleCommand.Flags().StringToStringVar(&namespaceSelectors, "namespace-selectors", nil, "Resources in the namespaces with the specified namespace labels will be migrated. All the labels provided in this option will be OR'ed")
 	createMigrationScheduleCommand.Flags().StringVarP(&clusterPair, "cluster-pair", "c", "", "Specify the name of the ClusterPair in the same namespace to be used for the migration")
+	createMigrationScheduleCommand.Flags().StringVar(&adminClusterPair, "admin-cluster-pair", "", "Specify the name of the admin ClusterPair used to migrate cluster-scoped resources, if the ClusterPair is present in a non-admin namespace")
 	createMigrationScheduleCommand.Flags().BoolVarP(&excludeResources, "exclude-resources", "", false, "If present, Kubernetes resources will not be migrated")
-	createMigrationScheduleCommand.Flags().BoolVarP(&excludeVolumes, excludeVolumesFlag, "", false, "If present, the underlying Portworx volumes will not be migrated. This is the only allowed and default behaviour in sync-dr use cases or if storage options are not provided in the cluster pair.")
+	createMigrationScheduleCommand.Flags().BoolVarP(&excludeVolumes, excludeVolumesFlag, "", false, "If present, the underlying Portworx volumes will not be migrated. This is the only allowed and default behaviour in sync-dr use-cases or if storage options are not provided in the cluster pair")
 	createMigrationScheduleCommand.Flags().BoolVarP(&startApplications, "start-applications", "a", false, "If present, the applications will be scaled up on the target cluster after a successful migration")
+	createMigrationScheduleCommand.Flags().BoolVar(&ignoreOwnerReferencesCheck, "ignore-owner-references-check", false, "If set, resources with ownerReferences will also be migrated, even if the corresponding owners are getting migrated")
+	createMigrationScheduleCommand.Flags().BoolVar(&purgeDeletedResources, "purge-deleted-resources", false, "Set this flag to automatically delete Kubernetes resources in the target cluster when they are removed from the source cluster")
+	createMigrationScheduleCommand.Flags().BoolVar(&skipServiceUpdate, "skip-service-update", false, "If set, service objects will be skipped during migration")
+	createMigrationScheduleCommand.Flags().BoolVar(&includeNetworkPolicyWithCIDR, "include-network-policy-with-cidr", false, "If set, the underlying network policies will be migrated even if a fixed CIDR is present on them")
+	createMigrationScheduleCommand.Flags().BoolVar(&disableSkipDeletedNamespaces, "disable-skip-deleted-namespaces", false, "If present, Stork will fail the migration when it encounters a namespace that is deleted but specified in the namespaces field. By default, Stork ignores deleted namespaces during migration")
```

Comment:
Are you referring to the `disable-skip-deleted-namespaces` flag?

The default behaviour is to skip deleted namespaces during migration. Which means that we will just ignore that deleted namespace and continue with the migration.

If we add this flag, in case a deleted namespace is given in the namespaces to migrate, the migration will fail.

If this was not clear, maybe we can modify the name of the flag. Any suggestions ??

Reply:
No it makes sense.